CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,12 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2024-01-31 Spell checking in Constellation
+<p>Spell checking in now available in text areas of Constellation Data access view (Extract Types and Extract Words Plugins) and Database importer. It can be added in other text areas as requested.
+When enabled, incorrect spellings and some grammar errors are highlighted and underlined. Left Clicking on a highlighted word/phrase will popup a suggestions list to choose from. An "Ignore All" button is available for spelling errors which will ignore the same error in the current text area.</p>
+<p>Constellation Option  "Enable spell checking in specified text areas" (Enabled by default) and the plugin Parameter option "setSpellCheckEnabled"(Disabled by default and should be enabled in the code per plugin parameter) are used to enable/disable spell checking.
```

Comment:
I would omit the plugin parameter option note since users won't actually see this. Instead maybe just refer fields where it has been allowed